website/docs/examples/controller_header/README.md

```diff
@@ -0,0 +1,165 @@
+---
+id: controller-info
+title: Controller 101
+---
+
+# Controller
+
+:::warning Danger
```

Comment:
Starting an example with red flags and danger doesn’t look good

Reply:
Resolved in commit [7a0b284](https://github.com/farm-ng/amiga-dev-kit/pull/212/commits/7a0b28462b3488a53c66381de77f6cb7645b684a)